src/connection/host_lookup.c

```diff
@@ -0,0 +1,591 @@
+/*
+ * Copyright 2008 Search Solution Corporation
+ * Copyright 2016 CUBRID Corporation
```

Comment:
```suggestion
 * Copyright 2016 CUBRID Corporation
```

Reply:
I can guess that "host_lookup.c" is written by me. So copyright belongs to me, doesn't it?

Reply:
LGTM :)